### PR TITLE
PR for mp3DirectCut-20.12

### PR DESCRIPTION
**Pull request by abdel792**
#### This Pull request is for the 20.12 version including the following fixes introduced in 20.11-dev:
    
* Stop speech during recording and reading for the latest versions of mp3directcut;
* Fixed reading remaining time for new versions of NVDA using Python 3.

### CODE REVIEW

**Review by nvdaes — COMMENTED** [review]
@Abdel792, though it's not strictly needed, you may want to accept my suggestion before we merge this PR, since when an add-on declares to be compatible with NVDA 2019.3, in fact its compatibility is extended to higher versions, since there aren't api changes with break compatibility with later versions. Anyway, I think this is understood and if you want we can merge this like that.
Also, I would suggest to use () before methods without a space before the first symbol (, since it's difficult to fi searched for screen reader users who review the add-on.

File: readme.md
Comment (nvdaes) [inline readme.md]:
```suggestion
* This add-on is compatible with the versions of NVDA ranging from 2016.4 until 2020.3 or higher.
```

**Review by nvdaes — CHANGES_REQUESTED** [review]
My suggestion is change compatibility line: instead of using "until 2020.3", you may use "until 2020.3 or higher", since when 2020.4 is released, users will find that the add-on is still compatible, due to the fact that api changes breaking compatibility aren't expected until NVDA 2021.1. Hope this helps.

**Comment by abdel792** [comment]
Hi @nvdaes,

Thank you.

I just created a new commit following your suggestion regarding compatible versions in readme.md.

**Review by nvdaes — APPROVED** [review]

**Comment by nvdaes** [comment]
I'll approve this and will merge into stable. In case it's useful, when someone requests changes sending a review on GitHub with suggestions produced adding comments for specific lines and pressing control+g, the author of the PR should have the ability to apply all suggestions searching an accessible button on GitHub, without needing to do an extra commit. This may especially useful when multiple changes are required.

**Comment by abdel792** [comment]
Hi @nvdaes,

Thank you very much for your approval.

**Comment by nvdaes** [comment]
Thank you @abdel792: I have merged this into stable branch too for svn.

**Pull request merged**
